Issue #2662: Change dependencies for cu_core and cu_atlas

alexfinnarn opened the pull request:
#2662

kreynen commented on modules/custom/cu_core/cu_core.info:
```diff
@@ -26,6 +26,7 @@ dependencies[] = xmlsitemap_node
 dependencies[] = xmlsitemap_menu
 dependencies[] = cu_helpscout
 dependencies[] = cu_inactive_users
+dependencies[] = cu_users
```
This can't be a real dependency because it causes secure_permission to get enabled which results in installs taking 10x as long.

https://github.com/CuBoulder/express/blob/dev/modules/custom/cu_users/cu_users.info#L6

alexfinnarn commented:
I added cu_users in both "prod" hosting modules. Since the dependency was in cu_atlas which is a dependency of cu_core, I would think installs would slow down since that change happened.

I also removed duplicate dependencies...concerning.

alexfinnarn commented:
Involves #2665

owenmorrill commented:
`Starting Drupal installation. This takes a while. Consider using the --notify global option.      [ok]
WD php: Warning: array_merge(): Argument #2 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).
WD php: Warning: array_merge(): Argument #1 is not an array in drupal_check_profile() (line 1189  [warning]
of /Users/.../sites/ld7/includes/install.inc).`
